packages/components/sidepanel/sidepanel-directives.ts

```diff
@@ -61,6 +64,12 @@ export class KbqSidepanelClose implements OnInit, OnChanges {
  */
 @Component({
     selector: 'kbq-sidepanel-header',
+    imports: [
+        KbqButton,
```

Comment:
Нужно подключать модуль, иначе сломаются стили

```suggestion
        KbqButtonModule,
```